### PR TITLE
add infra for test

### PR DESCRIPTION
**Pull request by chethanak**
https://github.com/d4rkr00t/prosemirror-dev-tools/issues/14

### CODE REVIEW

File: .idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```
Comment (d4rkr00t) [inline .idea/inspectionProfiles/Project_Default.xml]: Probably we should add them to ignore?

File: .babelrc [hunk 2]
```diff
@@ -1,12 +1,6 @@
...
   "presets": [
-    [
-      "env",
-      {
-        "modules": false
-      }
-    ],
-    "react"
+    "react",
+    "es2015"
```
Comment (d4rkr00t) [inline .babelrc:4]: What was wrong with "env"?

File: __tests__/components/button.test.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    theme: 'white05'
+  }];
+
+  it('Button renders when active', () => {
+    var button = renderer.create(<Button
+      params = {input}
+    />).toJSON();
+    expect(button).toMatchSnapshot();
+  });
+})
```
Comment (d4rkr00t) [inline __tests__/components/button.test.js]: Empty new line :)

File: __tests__/components/headings.test.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+  it('HeadingButton renders without crashing', () => {
+    var headingButton = renderer.create(<HeadingButton />).toJSON();
+    expect(headingButton).toMatchSnapshot();
+  });
+
+})
+
+
+
```
Comment (d4rkr00t) [inline __tests__/components/headings.test.js]: Too many new lines

File: __tests__/components/button.test.js (hunk 3 above)
Comment (d4rkr00t) [reply __tests__/components/button.test.js]: Interesting, prettier should fix that. Before you commit. I wonder why it didn't work for you.

File: .idea/inspectionProfiles/Project_Default.xml (hunk 1 above)
Comment (chethanak) [reply .idea/inspectionProfiles/Project_Default.xml]: yes will do

File: .babelrc (hunk 2 above)
Comment (chethanak) [reply .babelrc:4]: for jest its a requirement to add es2015

**Pull request closed without merging**